config/webpack.common.config.js

```diff
@@ -36,10 +36,6 @@ module.exports = {
         test: /\.(js|jsx)$/,
         exclude: /node_modules/,
         use: ['babel-loader']
-      },
-      {
```

Comment:
Have you considered using[ `mini-css-extract-plugin`](https://github.com/webpack-contrib/mini-css-extract-plugin)? I think it has a few advantages
- We do not need custom `prebuild` step and custom `buildStylesheets.js` script
- We can combine it with `less-loader`
- It should create separate CSS bundle of each entry point. 
- Each file can import the css it needs which I think is cleaner and we can move the css next to file instead of a separate style folder.

Reply:
I'm considering removing webpack bundles instead, so we don't bundle RDG's dependency in and simplify the workflow, so we wouldn't need that.

Reply:
We could also just move these files closer to their components, glob them, generate the index file and pass it to Less. Less only needs text as an input, so we don't actually *need* an index file.